benchbuild/projects/benchbuild/openssl.py

```diff
@@ -26,36 +21,47 @@
         "rc2test", "rc4test", "rmdtest", "sha1test", "sha256test", "sha512test",
         "shatest", "ssltest", "timingsafe", "utf8test"
     ]
+    SOURCE = [
+        HTTP(
+            remote={
+                '2.1.6.':
+                    'http://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.1.6.tar.gz'  # pylint: disable=line-too-long
```

Comment:
Codacy found an issue: [Useless suppression of 'line-too-long'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)